edx_sga/sga.py

```diff
@@ -165,9 +166,9 @@ def student_upload_max_size(cls):
         """
         returns max file size limit in system
         """
-        return getattr(
-            settings, "STUDENT_FILEUPLOAD_MAX_SIZE", cls.STUDENT_FILEUPLOAD_MAX_SIZE
-        )
+        return configuration_helpers.get_value('STUDENT_FILEUPLOAD_MAX_SIZE', getattr(
+             settings, "STUDENT_FILEUPLOAD_MAX_SIZE", cls.STUDENT_FILEUPLOAD_MAX_SIZE
```

Comment:
I think this has an extra space where should not be one and also please add the trailing comma.  @Nekenhei